spark/src/main/scala/org/apache/spark/sql/delta/coordinatedcommits/UCCommitCoordinatorBuilder.scala

```diff
@@ -252,3 +263,9 @@ object UCTokenBasedRestClientFactory extends UCClientFactory {
   override def createUCClient(uri: String, token: String): UCClient =
     new UCTokenBasedRestClient(uri, token)
 }
+
+/**
+ * Java-friendly holder for Unity Catalog configuration extracted from Spark configs.
```

Comment:
Let just avoid saying "Java-friendly holder", UCCatalogConfig actually is more readable than the tuples